product/HS/IntlAtlas/in/Services_Hierarchy.tsv

```diff
@@ -0,0 +1 @@
+code	name	level	name_en	name_es	name_short_en	name_short_es	parent_id
Services	Services	section	Services	Services	Services	Services	
Other	Other	2digit	Other	Other	Other	Other	0
Travel services	Travel services	2digit	Travel services	Travel services	Travel	Travel	0
Transport services	Transport services	2digit	Transport services	Transport services	Transport	Transport	0
Communications	Communications	2digit	Communications	Communications	Communications	Communications	0
Insurance and financial services	Insurance and financial services	2digit	Insurance and financial services	Insurance and financial services	Insurance and finance	Insurance and finance	0
Other	Other	4digit	Other	Other	Other	Other	1
Travel services	Travel services	4digit	Travel services	Travel services	Travel	Travel	2
Transport services	Transport services	4digit	Transport services	Transport services	Transport	Transport	3
Communications	Communications	4digit	Communications	Communications	Communications	Communications	4
Insurance and financial services	Insurance and financial services	4digit	Insurance and financial services	Insurance and financial services	Insurance and finance	Insurance and finance	5
```

Comment:
Not a big deal but looks like the services input files have weird line terminators (`\r` / `^M`), would be nice to switch to using `\n` like everything else.